tox_poetry_installer/hooks.py

```diff
@@ -117,6 +117,12 @@ def tox_addoption(parser: ToxParser):
         help="Automatically install all Poetry development dependencies to the environment",
     )
 
+    parser.add_testenv_attribute(
+        name="install_group_deps",
+        type="line-list",
+        help="Automatically install all Poetry dependencies belonging to specified groups to the environment",
```

Comment:
```suggestion
        help="List of Poetry dependency groups to install to the environment",
```
This language is more consistent with the `locked_deps` config option, which is the most similar existing option in the config.